### ISSUE TITLE
README error or typo ?

### ISSUE THREAD

**Issue by zw963** [opal/opal-rails#94]
When I read README, I see following section:

### Using Sprockets directives

If you want to use `application.js` (instead of `application.js.rb`) and keep Sprockets directives, you'll need to load the Opal files you require via Sprockets manually, e.g.:

```js
//= require opal
//= require opal_ujs
//= require turbolinks
//= require_tree .
//= require foobar

Opal.load('foobar');
```

The problem is:
why `` require foobar`` first, and then ``Opal.load(`foolbar`)``?

Whether i missing some special usage about `require` with `Opal.load` ??

Thanks

**Comment by elia** [comment]
These are the instruction for using `application.js`, if you instead prefer `application.js.rb` the syntax is just:

```rb
require "opal"
require "opal_ujs"
require "turbolinks"
require_tree "."
require "foobar"
```

I'll try to clarify it in the readme

**Comment by zw963** [comment]
@elia , Still not clear enough.

Should we must use ``//= require foobar`` and then to use
``Opal.load('foobar');`` ?

**Comment by elia** [comment]
Fixed by 0e6ddde
